src/Components/Employee.js

```diff
@@ -1,15 +1,22 @@
 import PetList from "./PetList";
 import "./Employee.css";
+import { formatEmployee } from "./utils/format";
+import { useState } from "react";
+
+
+export const Employee = ({firstName, lastName, prefix, postfix, title }) => {
+  const [showPets, setShowPets] = useState(false);
 
-export const Employee = () => {
   return (
     <article className="employee">
-      <h3>Staff Member Name</h3>
-      <h4>Staff Member Title</h4>
-      <button>Show Pets</button>
-      <PetList />
-    </article>
+      <h3>{formatEmployee(firstName,lastName, prefix, postfix, title)}</h3>
+      <h4>{title}</h4>
+      <button  onClick={() => {setShowPets((oldShowPets) => !oldShowPets);
```

Comment:
You're passing a callback to `setShowPets` instead of just toggling the `showPets` state